packages/editor/src/components/tabs/common/customfield/CustomFieldTable.tsx

```diff
@@ -2,14 +2,18 @@ import { CustomField, CustomFieldType } from '@axonivy/inscription-protocol';
 import { IvyIcons } from '@axonivy/editor-icons';
 import { ColumnDef, flexRender, getCoreRowModel, getSortedRowModel, SortingState, useReactTable } from '@tanstack/react-table';
 import { memo, useMemo, useState } from 'react';
-import { EditableCell } from './cell/EditableCell';
-import { Table } from './table/Table';
-import { SelectCell } from './cell/SelectCell';
-import { SelectItem } from '../select/Select';
-import { TableAddRow, TableFooter } from './footer/TableFooter';
-import { TableHeader, TableHeaderSorted } from './header/TableHeader';
-import { TableCell } from './cell/TableCell';
-import { ActionCell } from './cell/ActionCell';
+import {
+  SelectItem,
+  EditableCell,
+  SelectCell,
+  Table,
+  TableHeader,
+  TableHeaderSorted,
+  TableCell,
+  ActionCell,
+  TableFooter,
+  TableAddRow
+} from '../../../../components/widgets';
```

Comment:
is there a way in TS cluster similar import: so that we can just consume all 'table' widget features at once?